scripts/linux/3_pl_server.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-rm -rf /home/xilinx/pynq/bitstream/.log
```

Comment:
We already delete this file in start_pl_server.py. Are there any circumstances where we need to delete it again here?